content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
+
+For all other question types, once all of the information in both the question and answer has been given, but something needs to be fixed, the quizmaster will say "Again".
+
+##### Quote is Complete
+
+On a Situation type question, when the quizzer has completed the quotation, the quizmaster will say, "Quote is complete."
+
+##### Appending an Answer
+
+On an INT or MA question, if all the information in the answer text has been given but the quizzer is missing information in the question text, the quizmaster will say "More".
+
+On a Situation question, because the quotation is part of the question, the quizmaster will say "More" when required information from the quotation is missing.
+
+On a Quote or Reference question, if either the chapter or verse number were not completely finished when the quizzer jumped, the quizmaster will say "More".
 
 #### Commentary
-On a CVR/MACVR, in the scenario where the entire Reference (which is part of the "Question") was not read by the quizmaster, the quizmaster has two prompts to give, and must give them in appropriate order and with appropriate timing. If the quizzer provides all information in the question "text" and the answer "text", because the quizzer has not yet finished saying the complete Reference (part of the "question"), the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the "question" (which includes the Reference) and the answer.
+
+On a CVR/MACVR, in the scenario where the entire reference (which is part of the question text) was not read by the quizmaster, the quizmaster has 2 prompts to give, and must give them in the appropriate order and with appropriate timing. If the quizzer provides all information in the question text and the answer text, because the quizzer has not yet finished saying the complete reference, which is part of the question, the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the question (which includes the reference) and the answer.
 
 ### Answer Judges (AJ)
 
-- All conferring among quizmaster and answer judge(s) shall be done privately.
-    - When an answer given is in question, it is the answer judge who will determine the accuracy of its content and make a ruling accordingly.
-- Will watch the quizmaster's reading for accuracy.
-- Ensures the quizzer is recognized by the quizmaster when their light comes on.
+When an answer given is in question, the answer judge will determine the accuracy of its content and make a ruling recommendation to the quizmaster.
+
+Answer judges will also:
+
+- Watch the quizmaster's reading for accuracy
```

Comment:
"for validity"

Reply:
An AJ is checking for accuracy, not just validity.